### DISCUSSION TITLE
my trapsin keep opening potal only.

### DISCUSSION THREAD

**Discussion by SeanKR81** [isid0re/SoloLeveling#269] [Q&A]
I don't know why but I just changed to the newest version and my trapsin keeps opening his portal in some area(i can't specify yet)

I changed files after she met lvl 76.

**Comment by isid0re** [comment]
Any updates information on the areas?

**Reply by SeanKR81** [reply]
i can just guess it happens because of inventory..

it happens when there are many potions on the ground while I am carrying some.

**Reply by isid0re** [reply]
Then its portal to go to town to clear space. If you are using pickits those take space and will interfere with the bots progression.  Also depending on how old your install is, you might want to update your version to a newer version. Updated version uses less pots in the inventory
